dockerhub/distroless/Dockerfile

```diff
@@ -34,36 +33,35 @@ RUN apt-get update -qq \
       -fsSLO \
       --compressed \
       --retry 5 \
-      || (echo "error: unknown release: ($tag)" && exit 1) \
+      || (echo "error: failed to download: $tag" && exit 1) \
     && for key in \
       "F3DCC08A8572C0749B3E18888EAB4D40A7B22B59" \
     ; do \
       gpg --batch --keyserver hkps://keys.openpgp.org --recv-keys "$key" \
       || gpg --batch --keyserver keyserver.ubuntu.com --recv-keys "$key" ; \
     done \
-    && gpg --update-trustdb \
     && curl "https://github.com/oven-sh/bun/releases/$release/SHASUMS256.txt.asc" \
       -fsSLO \
       --compressed \
       --retry 5 \
     && gpg --batch --decrypt --output SHASUMS256.txt SHASUMS256.txt.asc \
-      || (echo "error: failed to verify release: ($tag)" && exit 1) \
+      || (echo "error: failed to verify: $tag" && exit 1) \
     && grep " bun-linux-$build.zip\$" SHASUMS256.txt | sha256sum -c - \
-      || (echo "error: failed to verify release: ($tag)" && exit 1) \
+      || (echo "error: failed to verify: $tag" && exit 1) \
     && unzip "bun-linux-$build.zip" \
     && mv "bun-linux-$build/bun" /usr/local/bin/bun \
     && rm -f "bun-linux-$build.zip" SHASUMS256.txt.asc SHASUMS256.txt \
     && chmod +x /usr/local/bin/bun \
-    && ln -s /usr/local/bin/bun /usr/local/bin/bunx \
     && which bun \
-    && which bunx \
     && bun --version
 
 FROM gcr.io/distroless/base-nossl-debian11
 
-COPY --from=build /usr/local/bin/bun /usr/local/bin
-COPY --from=build /usr/local/bin/bunx /usr/local/bin
+COPY --from=build /usr/local/bin/bun /usr/local/bin/
+
+# Known issue: `bunx` is not available in distroless.
+#
+# If `ln` is used in the build image, the size of the final
+# image will be double, because of: https://github.com/oven-sh/bun/issues/5269
```

Comment:
You missed my comment?: https://github.com/oven-sh/bun/issues/5269#issuecomment-1724856820

You can keep it:

```Dockerfile
# List of sources to destination (final path):
COPY --from=build \
  /usr/local/bin/bun /usr/local/bin/bunx \
  /usr/local/bin
```

Comparision:

```console
# Resolved with single COPY, symlink works:
$ docker save local-bun-build | gzip -c | wc -c | numfmt --to iec
40M

# The duplicate issue that previously existed:
$ docker save oven/bun | gzip -c | wc -c | numfmt --to iec
97M

# Resolved vs original image size:
$ docker images
REPOSITORY                  TAG                IMAGE ID       CREATED          SIZE
local-bun-build             latest             020acf83835e   7 minutes ago    110MB
oven/bun                    latest             647ebb4444b7   3 days ago       271MB
```